docs/source/config.json

```diff
@@ -62,9 +62,17 @@
     "API Reference": {
       "ApolloServer": "/api/apollo-server",
       "startStandaloneServer": "/api/standalone",
-      "expressMiddleware": "/api/express-middleware",
-      "@apollo/subgraph": "https://www.apollographql.com/docs/federation/api/apollo-subgraph/",
-      "@apollo/gateway": "https://www.apollographql.com/docs/federation/api/apollo-gateway/"
+      "expressMiddleware": "/api/express-middleware"
+    },
+    "Using with Federation": {
+      "As a subgraph": {
+        "Setup": "/using-federation/apollo-subgraph-setup",
+        "@apollo/subgraph": "/using-federation/api/apollo-subgraph/"
```

Comment:
```suggestion
        "@apollo/subgraph reference": "/using-federation/api/apollo-subgraph/"
```

Reply:
Because these two articles aren't under an "API Reference" header anymore, probably makes sense to include the word "reference" in the nav item